src/register_target.cpp

```diff
@@ -86,16 +84,5 @@ target make_target(const std::string& name)
     return it->second;
 }
 
-std::vector<std::string> get_targets()
```

Comment:
This function should not be removed.

Reply:
Something is not right; I reverted that change. I will double-check.

Reply:
I fixed it.